trans_sec/packet/inspect_layer.py

```diff
@@ -71,7 +82,7 @@ class IntMeta2(IntMeta):
     name = "INT_META_2"
 
 
-class IntMeta3(IntMeta):
+class SourceIntMeta(SourceMeta):
     """
     This class represents the third INT metadata being placed onto the packets
     """
```

Comment:
the only other think I can think to do to improve would be to change the name of the 'new' class SourceIntMeta to something other than "INT_META_3"